packages/api-client/src/api-client.ts

```diff
@@ -239,10 +241,10 @@ export class GFWAPI {
         const data = await fetch(fetchUrl, {
           method,
           signal,
-          ...(body && { body: JSON.stringify(body) }),
+          ...(requestType === 'json' ? body && { body: JSON.stringify(body) } : body && { body }),
```

Comment:
The objective of the spread operator in the body is to include the property only when the body is defined, to understand how it works take a look at: https://medium.com/@mikeh91/conditionally-adding-keys-to-javascript-objects-using-spread-operators-and-short-circuit-evaluation-acf157488ede
With this change, at least for me, it harder to understand what is happening so my suggestion to simplify it is:
```suggestion
          ...body && { body: requestType === 'json' ? JSON.stringify(body): body },
```

Reply:
Thanks for the change, LGTM!